src/hooks/useFilterCriteria.ts

```diff
@@ -46,10 +47,7 @@ export const useFilterCriteria = (): useFilterCriteriaReturns => {
         ).totalSupply;
 
         return (
-          (proposal.stateInVotingMachine ===
-            VotingMachineProposalState.QuietEndingPeriod ||
-            proposal.stateInVotingMachine ===
-              VotingMachineProposalState.Queued) &&
+          proposal.stateInVotingMachine === VotingMachineProposalState.Queued &&

```

Comment:
@hhamud  The criteria was coming from a conversation with @AugustoL and @rossneilson on the issue when adding these filters. 
Was the initial criteria. If the issue is for Quiet Ending Period sorting wrong, this wouldn't be the problem. 
I am not sure of the origin of this issue, but lets keep a conversation with the guys to be sure we don't break the initial set up for the filtering of the first render.